src/Manager.sol

```diff
@@ -150,6 +148,11 @@ abstract contract Manager is
         _pause();
     }
 
+    /// @notice Returns the address of the owner of this contract.
+    function getOwner() external view returns (address) {
```

Comment:
Any particular reason you've wrapped `owner()` here? Unless I'm missing something I would prefer without the wrapping

Reply:
couldn't access owner() directly by just importing the interface

Reply:
Ah can you instead add an interface for OwnableUpgradeable and make sure it's accessible that way pls?

Reply:
I still dislike this pattern quite a lot :( The fix is to just wrap the manager address in `getManagerOwner` with the `IOwnable` interface and call `owner()` directly

Reply:
ah yea just haven't pushed this change